ydb/topic.py

```diff
@@ -337,7 +337,7 @@ def writer(
         encoders: Optional[Mapping[_ydb_topic_public_types.PublicCodec, Callable[[bytes], bytes]]] = None,
         encoder_executor: Optional[concurrent.futures.Executor] = None,  # default shared client executor pool
     ) -> TopicWriter:
-        args = locals()
+        args = locals().copy()
```

Comment:
why you make a copy?

Reply:
because locals() return dynamic dict with infinite recursive field "args" and I can't remove it at next line.